sidekiq-scheduler.gemspec

```diff
@@ -30,4 +30,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'activejob'
   s.add_development_dependency 'rack-test'
   s.add_development_dependency 'rack', '< 3'
+  s.add_development_dependency 'rack-session'
```

Comment:
Sidekiq web in 8 requires Rack 3 which is its own can of worms. The session stuff was extracted into a separate gem so I figure may as well include it now.